ui/logviewer/App.jsx

```diff
@@ -214,6 +223,23 @@ class App extends React.PureComponent {
     }
   };
 
+  logFormatter = (log) => {
```

Comment:
Also change `log` to `line` so it's clearer that it's one line at a time passed in as an arg.